### PR TITLE
Interpreter state chunk fixes

### PR DESCRIPTION
**Pull request by fmatthew5876**
Saves space and removes ugly code from https://github.com/EasyRPG/Player/pull/1739

### CODE REVIEW

File: generator/csv/fields.csv [hunk 1]
```diff
@@ -936,7 +936,7 @@ SaveEventExecState,keyinput_left,f,Boolean,0x24,False,0,0,
 SaveEventExecState,keyinput_right,f,Boolean,0x25,False,0,0,
 SaveEventExecState,keyinput_up,f,Boolean,0x26,False,0,0,
 SaveEventExecState,keyinput_timed,f,Boolean,0x29,False,0,0,
-SaveEventExecState,unused_wait_for_key_or_enter,f,Boolean,0x2a,False,0,0,If enabled; an event waits for either the confirmation key to be pressed or one of the keys defined by KeyInputProc before continuing. This flag seems to be unused though since it is never written to (keyinput_wait is used instead).
```
Comment (fmatthew5876) [inline generator/csv/fields.csv]: Whats up with this comment? I tested with rm2k3e and it definately uses this chunk when you do a "Wait for Key Input" command. That is the wait command variant that waits for input, not a key input proc.

File: generator/csv/fields.csv [hunk 2]
```diff
@@ -936,7 +936,7 @@ SaveEventExecState,keyinput_left,f,Boolean,0x24,False,0,0,
...
+SaveEventExecState,wait_key_enter,f,Boolean,0x2a,False,0,0,Used for a wait command "Wait For Key Input" rm2k3 feature to wait for decsion key press.
```
Comment (carstene1ns) [inline generator/csv/fields.csv]: > decision

Comment (fmatthew5876) [reply generator/csv/fields.csv]: Fixed, thanks!

**Comment by fmatthew5876** [comment]
I took out the `keyinput_variable` fix from here. It breaks #155 and https://github.com/EasyRPG/Player/issues/1674

We need a better solution for that, but we should consider it separately

**Comment by Ghabry** [comment]
This keyinput variable naming is really ugly :/
Different devs for 2k Value! and 2k3.

How does 2k before Value! (1.50) store key input?

**Comment by fmatthew5876** [comment]
It's horrible I agree but I don't know a better solution. With the old names I had to add comments in my PR explaining why I was assigning direction to shift. With the new ugly names, no comments needed.

Old rm2k uses only all directions, decision, and cancel chunks.

**Pull request merged**